src/test/kotlin/CalculateTest.kt

```diff
@@ -0,0 +1,11 @@
+import org.junit.jupiter.api.Test
+
+class SumTest {
+
+  @Test
+  fun testCalculate() {
+    val sum = Sum()
+    val result = sum.calculate()
```

Comment:
This test miss an assertion. What is the expectation -- what should result equal to?